src/headless/celery_app.py

```diff
@@ -1,10 +1,11 @@
 # coding=utf-8
 import os
+
 from celery import Celery
 
+from headless.settings import OUTPUT_DIRECTORY
```

Comment:
Shouldn't we import from 3rd party first before the project's code?

Reply:
:D. I just use PyCharm optimize imports. It seems to be sorting alphabetically.